lab/src/main/kotlin/materialui/lab/components/treeItem/treeItem.kt

```diff
@@ -0,0 +1,35 @@
+package materialui.lab.components.treeItem
+
+import kotlinx.html.DIV
+import kotlinx.html.Tag
+import kotlinx.html.TagConsumer
+import materialui.components.StandardProps
+import materialui.components.toolbar.enums.ToolbarStyle
+import react.*
+
+@JsModule("@material-ui/lab/TreeItem")
+private external val treeItemModule: dynamic
+
+external interface TreeItemProps : StandardProps {
+    var collapseIcon: ReactElement?
+    var endIcon: ReactElement?
+    var expandIcon: ReactElement?
+    var icon: ReactElement?
+    var label: ReactElement?
+    var nodeId: String?
+    var TransitionComponent: dynamic
+}
+
+@Suppress("UnsafeCastFromDynamic")
+private val treeItemComponent: RClass<TreeItemProps> = treeItemModule.default
+
+fun RBuilder.treeItem(
+    vararg classMap: Pair<ToolbarStyle, String>,
+    block: TreeItemElementBuilder<DIV>.() -> Unit
```

Comment:
The component of TreeItem is `li`. So, please specify LI instead of DIV.

https://github.com/mui-org/material-ui/blob/024e176b9d2d0dc96826b0fec630823254372683/packages/material-ui-lab/src/TreeItem/TreeItem.js#L371-L384